packages/core/src/utils/__tests__/regex.test.ts

```diff
@@ -0,0 +1,35 @@
+import { credDefIdRegex, issuerDidRegex, schemaIdRegex, schemaIssuerDidRegex, schemaVersionRegex } from '../regex'
+
+describe('Valid Regular Expression', () => {
+  const invalidTest = 'test'
+
+  test('test for credDefIdRegex', async () => {
+    const test = 'q7ATwTYbQDgiigVijUAej:3:CL:160971:1.0.0'
+    expect(test).toMatch(credDefIdRegex)
+    expect(credDefIdRegex.test(invalidTest)).toBeFalsy
```

Comment:
Will this work without `()`? If you do `toBeTruthy` will that throw an error?

```suggestion
    expect(credDefIdRegex.test(invalidTest)).toBeFalsy()
```

Reply:
Bot are fine! I just didn't know you could call it without the `()` :)